### PR TITLE
abstracting logic for filter colors

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
🌈

### CODE REVIEW

File: src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx [hunk 1]
```diff
@@ -383,8 +386,6 @@ class SearchFiltersContainer extends Component {
 
     const apContainerClass = 'ap-container';
     const commonContainerClass = tandemIsSelected ? 'tandem-common-filters' : '';
-    const tandem1Class = 'tandem-1-filters';
-    const tandem2Class = 'tandem-2-filters';
```
Comment (elizabeth-jimenez) [inline src/Components/SearchFilters/SearchFiltersContainer/SearchFiltersContainer.jsx:387]: I removed these cause they never seem to get set. The `tandem-1-filters` and `tandem-2-filters` seemed to only get set in a parent div using `tandemUserClass `

File: src/Components/Toggle/__snapshots__/Toggle.test.jsx.snap [hunk 2]
```diff
@@ -5,23 +5,23 @@ exports[`Toggle matches snapshot 1`] = `
...
+    className="toggle-button toggle-button--selected undefined"
     key="1"
     onClick={[Function]}
     value={1}
   >
     a
   </button>
   <button
-    className="toggle-button  toggle-ap"
+    className="toggle-button  undefined"
```
Comment (mjoyce91) [inline src/Components/Toggle/__snapshots__/Toggle.test.jsx.snap:16]: check out this coming through as undefined

Comment (elizabeth-jimenez) [reply src/Components/Toggle/__snapshots__/Toggle.test.jsx.snap:16]: 🙈

File: src/Components/Toggle/Toggle.test.jsx [hunk 3]
```diff
@@ -62,6 +66,17 @@ describe('Toggle', () => {
...
+    const wrapper = shallow(
+      <Toggle {...props} />,
+    );
+    expect(wrapper.find('button').at(0).hasClass('toggle-tandem-1')).toBe(true);
+    expect(wrapper.find('button').at(1).hasClass('toggle-tandem-2')).toBe(true);
+    expect(wrapper.find('button').at(2).hasClass('toggle-pv')).toBe(true);
+    expect(wrapper.find('button').at(3).hasClass('toggle-ap')).toBe(true);
+    expect(wrapper.find('button').at(4).hasClass('toggle-ap')).toBe(false);
+  });
+
```
Comment (mjoyce91) [inline src/Components/Toggle/Toggle.test.jsx:79]: good tests 👍

Comment (elizabeth-jimenez) [reply src/Components/Toggle/Toggle.test.jsx:79]: i learned from your PR i reviewed a few weeks ago 🥇

**Pull request merged**